coalib/misc/Exceptions.py

```diff
@@ -15,6 +17,12 @@ def get_exitcode(exception, log_printer=None):
         exitcode = 0
     elif isinstance(exception, SystemExit):
         exitcode = exception.code
+    elif (isinstance(exception, ContextualVersionConflict) or
```

Comment:
You can simply do `isinstance(exception, (VersionConflict, ContextualVersionConflict))`
I think